2020-11-12 10:49:44.258 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.hzl.service.UserService' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@javax.annotation.Resource(shareable=true, lookup=, name=, description=, authenticationType=CONTAINER, type=class java.lang.Object, mappedName=)}
2020-11-12 10:50:23.930 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.hzl.service.UserService' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@javax.annotation.Resource(shareable=true, lookup=, name=, description=, authenticationType=CONTAINER, type=class java.lang.Object, mappedName=)}
2020-11-12 10:53:15.703 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.hzl.service.UserService' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@javax.annotation.Resource(shareable=true, lookup=, name=, description=, authenticationType=CONTAINER, type=class java.lang.Object, mappedName=)}
2020-11-12 10:56:45.487 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.hzl.service.UserService' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@javax.annotation.Resource(shareable=true, lookup=, name=, description=, authenticationType=CONTAINER, type=class java.lang.Object, mappedName=)}
2020-11-12 11:00:34.824 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.hzl.service.UserService' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@javax.annotation.Resource(shareable=true, lookup=, name=, description=, authenticationType=CONTAINER, type=class java.lang.Object, mappedName=)}
2020-11-12 11:07:05.744 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'areaDicMapper' and 'com.hzl.mapper.AreaDicMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.746 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'commentMapper' and 'com.hzl.mapper.CommentMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.747 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'hotelExtendPropertyMapper' and 'com.hzl.mapper.HotelExtendPropertyMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.747 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'hotelFeatureMapper' and 'com.hzl.mapper.HotelFeatureMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.747 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'hotelMapper' and 'com.hzl.mapper.HotelMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.747 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'hotelOrderMapper' and 'com.hzl.mapper.HotelOrderMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.747 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'hotelRoomMapper' and 'com.hzl.mapper.HotelRoomMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.748 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'hotelTempStoreMapper' and 'com.hzl.mapper.HotelTempStoreMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.748 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'hotelTradingAreaMapper' and 'com.hzl.mapper.HotelTradingAreaMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.748 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'imageMapper' and 'com.hzl.mapper.ImageMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.748 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'labelDicMapper' and 'com.hzl.mapper.LabelDicMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.748 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'orderLinkUserMapper' and 'com.hzl.mapper.OrderLinkUserMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.748 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'productStoreMapper' and 'com.hzl.mapper.ProductStoreMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.748 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'tradeEndsMapper' and 'com.hzl.mapper.TradeEndsMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.748 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'userLinkUserMapper' and 'com.hzl.mapper.UserLinkUserMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.749 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - Skipping MapperFactoryBean with name 'userMapper' and 'com.hzl.mapper.UserMapper' mapperInterface. Bean already defined with the same name!
2020-11-12 11:07:05.749 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - No MyBatis mapper was found in '[com.hzl.mapper]' package. Please check your configuration.
2020-11-12 11:07:06.497 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.hzl.service.UserService' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@javax.annotation.Resource(shareable=true, lookup=, name=, description=, authenticationType=CONTAINER, type=class java.lang.Object, mappedName=)}
2020-11-12 11:24:09.172 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - No MyBatis mapper was found in '[com.hzl.itripauth]' package. Please check your configuration.
2020-11-12 11:32:39.309 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - No MyBatis mapper was found in '[com.hzl.itripauth]' package. Please check your configuration.
2020-11-12 11:32:51.615 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Lookup method resolution failed; nested exception is java.lang.IllegalStateException: Failed to introspect Class [com.hzl.itripauth.controller.UserController] from ClassLoader [org.springframework.boot.devtools.restart.classloader.RestartClassLoader@7514c4fc]
2020-11-12 11:57:19.907 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userServiceImpl': Injection of autowired dependencies failed; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'email.send.enable' in value "${email.send.enable}"
2020-11-12 11:58:13.340 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userServiceImpl': Injection of autowired dependencies failed; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'email.send.enable' in value "${email.send.enable}"
2020-11-12 11:59:40.362 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userServiceImpl': Injection of autowired dependencies failed; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'email.send.enable' in value "${email.send.enable}"
2020-11-12 14:18:47.442 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Lookup method resolution failed; nested exception is java.lang.IllegalStateException: Failed to introspect Class [com.hzl.itripauth.controller.UserController] from ClassLoader [org.springframework.boot.devtools.restart.classloader.RestartClassLoader@4156e1fa]
2020-11-12 14:46:18.762 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Lookup method resolution failed; nested exception is java.lang.IllegalStateException: Failed to introspect Class [com.hzl.itripauth.controller.UserController] from ClassLoader [org.springframework.boot.devtools.restart.classloader.RestartClassLoader@5948317b]
2020-11-12 14:46:55.960 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Lookup method resolution failed; nested exception is java.lang.IllegalStateException: Failed to introspect Class [com.hzl.itripauth.controller.UserController] from ClassLoader [org.springframework.boot.devtools.restart.classloader.RestartClassLoader@bc03d18]
2020-11-12 14:57:22.482 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - No MyBatis mapper was found in '[com.hzl.itripauth]' package. Please check your configuration.
2020-11-12 14:57:22.744 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is org.springframework.boot.autoconfigure.jdbc.DataSourceProperties$DataSourceBeanCreationException: Failed to determine a suitable driver class
2020-11-12 14:57:30.743 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userServiceImpl': Injection of autowired dependencies failed; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'email.send.enable' in value "${email.send.enable}"
2020-11-12 15:01:04.297 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userController': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userServiceImpl': Injection of autowired dependencies failed; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'email.send.enable' in value "${email.send.enable}"
2020-11-12 15:31:17.858 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - No MyBatis mapper was found in '[com.hzl.itripauth]' package. Please check your configuration.
2020-11-12 15:31:18.857 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'handlerExceptionResolver' defined in class path resource [org/springframework/boot/autoconfigure/web/servlet/WebMvcAutoConfiguration$EnableWebMvcConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.web.servlet.HandlerExceptionResolver]: Factory method 'handlerExceptionResolver' threw exception; nested exception is java.lang.IllegalStateException: Ambiguous @ExceptionHandler method mapped for [class java.lang.Exception]: {public com.hzl.common.vo.RetrunResult com.handler.GlobalExceptionHandler.error(com.hzl.exception.SysException), public com.hzl.common.vo.RetrunResult com.handler.GlobalExceptionHandler.error(com.hzl.exception.DaoException)}
2020-11-12 15:38:43.335 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - No MyBatis mapper was found in '[com.hzl.itripauth]' package. Please check your configuration.
2020-11-12 15:38:44.942 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'handlerExceptionResolver' defined in class path resource [org/springframework/boot/autoconfigure/web/servlet/WebMvcAutoConfiguration$EnableWebMvcConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.web.servlet.HandlerExceptionResolver]: Factory method 'handlerExceptionResolver' threw exception; nested exception is java.lang.IllegalStateException: Ambiguous @ExceptionHandler method mapped for [class java.lang.Exception]: {public com.hzl.common.vo.RetrunResult com.hzl.itripauth.handler.GlobalExceptionHandler.error(com.hzl.exception.SysException), public com.hzl.common.vo.RetrunResult com.hzl.itripauth.handler.GlobalExceptionHandler.error(com.hzl.exception.DaoException)}
2020-11-12 16:02:38.788 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - No MyBatis mapper was found in '[com.hzl.itripauth]' package. Please check your configuration.
2020-11-12 16:06:44.970 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - No MyBatis mapper was found in '[com.hzl.itripauth]' package. Please check your configuration.
2020-11-12 16:17:53.886 [http-nio-8801-exec-1] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.web.HttpRequestMethodNotSupportedException: Request method 'POST' not supported]
2020-11-12 16:17:56.867 [http-nio-8801-exec-3] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:18:26.211 [http-nio-8801-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.web.HttpRequestMethodNotSupportedException: Request method 'POST' not supported]
2020-11-12 16:18:28.662 [http-nio-8801-exec-5] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:19:47.476 [http-nio-8801-exec-8] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:21:13.412 [http-nio-8801-exec-1] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:21:16.042 [http-nio-8801-exec-3] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:22:11.530 [http-nio-8801-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:22:13.221 [http-nio-8801-exec-5] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:22:53.956 [http-nio-8801-exec-6] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:25:05.256 [http-nio-8801-exec-9] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.web.HttpMediaTypeNotSupportedException: Content type 'multipart/form-data;boundary=--------------------------113355478221326329517651;charset=UTF-8' not supported]
2020-11-12 16:26:55.892 [restartedMain] WARN  org.mybatis.spring.mapper.ClassPathMapperScanner - No MyBatis mapper was found in '[com.hzl.itripauth]' package. Please check your configuration.
2020-11-12 16:27:46.610 [http-nio-8801-exec-2] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.web.HttpMediaTypeNotSupportedException: Content type 'multipart/form-data;boundary=--------------------------321799237634328427088766;charset=UTF-8' not supported]
2020-11-12 16:28:02.290 [http-nio-8801-exec-3] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.web.HttpMediaTypeNotSupportedException: Content type 'multipart/form-data;boundary=--------------------------956271660937286314315550;charset=UTF-8' not supported]
2020-11-12 16:28:10.859 [http-nio-8801-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:29:02.791 [http-nio-8801-exec-5] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:30:08.068 [http-nio-8801-exec-8] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:30:13.999 [http-nio-8801-exec-9] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public com.hzl.common.vo.RetrunResult com.hzl.itripauth.controller.UserController.doRegister(com.hzl.itripauth.condition.UserRegisterCondition)]
2020-11-12 16:32:24.092 [http-nio-8801-exec-2] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.web.HttpMediaTypeNotSupportedException: Content type 'multipart/form-data;boundary=--------------------------858796821651732800866774;charset=UTF-8' not supported]
2020-11-12 16:32:46.257 [http-nio-8801-exec-3] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.web.HttpMediaTypeNotSupportedException: Content type 'multipart/form-data;boundary=--------------------------963347357535355957178304;charset=UTF-8' not supported]
2020-11-12 16:33:44.948 [http-nio-8801-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.web.HttpMediaTypeNotSupportedException: Content type 'application/x-www-form-urlencoded;charset=UTF-8' not supported]
2020-11-12 16:33:50.630 [http-nio-8801-exec-5] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.web.HttpMediaTypeNotSupportedException: Content type 'application/x-www-form-urlencoded;charset=UTF-8' not supported]
2020-11-12 16:34:09.309 [http-nio-8801-exec-6] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.web.HttpMediaTypeNotSupportedException: Content type 'text/plain;charset=UTF-8' not supported]
2020-11-12 16:35:57.307 [http-nio-8801-exec-9] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
### The error may exist in com/hzl/mapper/UserMapper.java (best guess)
### The error may involve com.hzl.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO itrip_user  ( user_code, user_password, user_type,       creation_date,  modify_date )  VALUES  ( ?, ?, ?,       ?,  ? )
### Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
; Field 'user_name' doesn't have a default value; nested exception is java.sql.SQLException: Field 'user_name' doesn't have a default value]
2020-11-12 16:36:06.894 [http-nio-8801-exec-10] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
### The error may exist in com/hzl/mapper/UserMapper.java (best guess)
### The error may involve com.hzl.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO itrip_user  ( user_code, user_password, user_type,       creation_date,  modify_date )  VALUES  ( ?, ?, ?,       ?,  ? )
### Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
; Field 'user_name' doesn't have a default value; nested exception is java.sql.SQLException: Field 'user_name' doesn't have a default value]
2020-11-12 16:36:16.156 [http-nio-8801-exec-1] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
### The error may exist in com/hzl/mapper/UserMapper.java (best guess)
### The error may involve com.hzl.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO itrip_user  ( user_code, user_password, user_type,       creation_date,  modify_date )  VALUES  ( ?, ?, ?,       ?,  ? )
### Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
; Field 'user_name' doesn't have a default value; nested exception is java.sql.SQLException: Field 'user_name' doesn't have a default value]
2020-11-12 16:37:32.824 [http-nio-8801-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.web.HttpRequestMethodNotSupportedException: Request method 'POST' not supported]
2020-11-12 16:37:41.010 [http-nio-8801-exec-5] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-11-12 16:37:46.069 [http-nio-8801-exec-6] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-11-12 16:38:17.835 [http-nio-8801-exec-7] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
### The error may exist in com/hzl/mapper/UserMapper.java (best guess)
### The error may involve com.hzl.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO itrip_user  ( user_code, user_password, user_type,       creation_date,  modify_date )  VALUES  ( ?, ?, ?,       ?,  ? )
### Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
; Field 'user_name' doesn't have a default value; nested exception is java.sql.SQLException: Field 'user_name' doesn't have a default value]
2020-11-12 16:38:23.071 [http-nio-8801-exec-8] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
### The error may exist in com/hzl/mapper/UserMapper.java (best guess)
### The error may involve com.hzl.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO itrip_user  ( user_code, user_password, user_type,       creation_date,  modify_date )  VALUES  ( ?, ?, ?,       ?,  ? )
### Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
; Field 'user_name' doesn't have a default value; nested exception is java.sql.SQLException: Field 'user_name' doesn't have a default value]
2020-11-12 21:12:32.690 [http-nio-8801-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.transaction.CannotCreateTransactionException: Could not open JDBC Connection for transaction; nested exception is com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.]
2020-11-12 21:13:11.217 [http-nio-8801-exec-7] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
### The error may exist in com/hzl/mapper/UserMapper.java (best guess)
### The error may involve com.hzl.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO itrip_user  ( user_code, user_password, user_type,       creation_date,  modify_date )  VALUES  ( ?, ?, ?,       ?,  ? )
### Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
; Field 'user_name' doesn't have a default value; nested exception is java.sql.SQLException: Field 'user_name' doesn't have a default value]
2020-11-12 21:13:44.841 [http-nio-8801-exec-8] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
### The error may exist in com/hzl/mapper/UserMapper.java (best guess)
### The error may involve com.hzl.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO itrip_user  ( user_code, user_password, user_type,       creation_date,  modify_date )  VALUES  ( ?, ?, ?,       ?,  ? )
### Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
; Field 'user_name' doesn't have a default value; nested exception is java.sql.SQLException: Field 'user_name' doesn't have a default value]
2020-11-12 21:14:04.410 [http-nio-8801-exec-1] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
### The error may exist in com/hzl/mapper/UserMapper.java (best guess)
### The error may involve com.hzl.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO itrip_user  ( user_code, user_password, user_type,       creation_date,  modify_date )  VALUES  ( ?, ?, ?,       ?,  ? )
### Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
; Field 'user_name' doesn't have a default value; nested exception is java.sql.SQLException: Field 'user_name' doesn't have a default value]
2020-11-12 21:19:10.248 [HikariPool-1 housekeeper] WARN  com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=3m48s709ms758µs500ns).
2020-11-12 21:19:10.261 [http-nio-8801-exec-2] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
### The error may exist in com/hzl/mapper/UserMapper.java (best guess)
### The error may involve com.hzl.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO itrip_user  ( user_code, user_password, user_type,       creation_date,  modify_date )  VALUES  ( ?, ?, ?,       ?,  ? )
### Cause: java.sql.SQLException: Field 'user_name' doesn't have a default value
; Field 'user_name' doesn't have a default value; nested exception is java.sql.SQLException: Field 'user_name' doesn't have a default value]
2020-11-12 21:23:47.233 [HikariPool-1 housekeeper] WARN  com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=45s392ms685µs100ns).
2020-11-12 21:24:31.712 [http-nio-8801-exec-2] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.mail.MailAuthenticationException: Authentication failed; nested exception is javax.mail.AuthenticationFailedException: 535 Error: authentication failed
]
